osmomath/decimal.go

```diff
@@ -582,6 +582,12 @@ func BigDecFromDec(d Dec) BigDec {
 	return NewBigDecFromBigIntWithPrec(d.BigInt(), PrecisionDec)
 }
 
+// BigDecFromSDKInt returns the BigDec representation of an sdkInt.
```

Comment:
nit: I think we do not reference sdk anymore, Int is defined in osmomath package
```suggestion
// BigDecFromSDKInt returns the BigDec representation of an Int.
```